app/scripts/main.js

```diff
@@ -41,6 +40,7 @@ import 'scripts/directives/pickadate.js';
 import 'scripts/directives/registrationTypeSelect.js';
 import 'scripts/directives/selectOnClick.js';
 import 'scripts/directives/showAnswer.js';
+import 'scripts/directives/showGroup.js';
```

Comment:
Sorry this file is still around. That's annoying. I've had other apps go search for all JS files when I was doing migration to webpack. I sorta remember scanning for all js files didn't work but I can't remember why.

The ideal way to do it is import the JS file you need in the component where it is used. You could do that... and then you wouldn't have to list it here. You could also put the modal component in the same folder as the parent component if that's the only component that uses it. This app has a very flat directory structure which I think makes it hard to find files. I'd prefer files to be grouped by screen and/or feature.

I guess these are just comments for improving the repo in the future. You don't necessarily need to address them in this PR.